src/designer/elsa-workflows-studio/src/components/editors/properties/elsa-script-property/elsa-script-property.tsx

```diff
@@ -58,6 +79,23 @@ export class ElsaScriptProperty {
     const fieldLabel = propertyDescriptor.label || propertyName;
     const fieldHint = propertyDescriptor.hint;
     const value = this.currentValue;
+    const isSchema = fieldName === "Schema";
```

Comment:
"Schema" is a property specific to HTTP Endpoint, which means it doesn't belong here.